core/app/models/spree/shipping_category.rb

```diff
@@ -7,5 +7,18 @@ class ShippingCategory < Spree::Base
       has_many :shipping_method_categories
     end
     has_many :shipping_methods, through: :shipping_method_categories
+    
+    before_save :clear_cache
+
+    def self.default
+      Rails.cache.fetch("#{Rails.application.class.parent_name.underscore}_default_shipping_category") do
+        first
```

Comment:
Will this behave correctly in case there are lazy loads already triggered from the resource? In case these lazy loaded referenced objects are also persisted in the cache, and the cache is not busted on writes to these you could get inconsistent state on reads.